Controller/Adminhtml/Menu/DownloadImportSample.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Snowdog\Menu\Controller\Adminhtml\Menu;
+
+use Magento\Backend\App\Action;
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\App\Response\Http\FileFactory as HttpFileFactory;
+use Psr\Log\LoggerInterface;
+use Snowdog\Menu\Model\Menu\Import\SampleFile as ImportSampleFile;
+
+class DownloadImportSample extends Action
+{
+    /**
+     * @inheritDoc
+     */
+    const ADMIN_RESOURCE = 'Snowdog_Menu::menus';
+
+    const FILE_NAME = 'menu-sample.csv';
+
+    /**
+     * @var HttpFileFactory
+     */
+    private $httpFileFactory;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var ImportSampleFile
+     */
+    private $importSampleFile;
+
+    public function __construct(
+        Action\Context $context,
+        HttpFileFactory $httpFileFactory,
+        LoggerInterface $logger,
+        ImportSampleFile $importSampleFile
+    ) {
+        $this->httpFileFactory = $httpFileFactory;
+        $this->logger = $logger;
+        $this->importSampleFile = $importSampleFile;
+
+        parent::__construct($context);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function execute()
```

Comment:
I do not see that we provide any sample file (nor create one). Also it looks like it should be in {magentoRoot}/var and as far as I know we do not have to move file there via composer.json instructions

Reply:
ok, found code that generates sample file ...